x-pack/plugins/security/server/audit/audit_service.ts

```diff
@@ -19,6 +19,8 @@ import { SpacesPluginSetup } from '../../../spaces/server';
 import { AuditEvent, httpRequestEvent } from './audit_events';
 import { SecurityPluginSetup } from '..';
 
+export const ECS_VERSION = '1.6.0';
```

Comment:
Mostly unrelated, but can we also [update this comment](https://github.com/elastic/kibana/blob/f413957827b0294d12cf9e460d73c6ff26314fe8/x-pack%2Fplugins%2Fsecurity%2Fserver%2Faudit%2Faudit_events.ts#L12) to indicate that we're targeting 1.6 instead of 1.5?

Reply:
Good catch!